src/NHibernate.Test/NHSpecificTest/GH1439/FixtureByCode.cs

```diff
@@ -0,0 +1,78 @@
+﻿using NHibernate.Test.NHSpecificTest.GH0000;
```

Comment:
Please do not keep a dependency on the "template" bug. (It seems removable without any other changes here by the way.)